docs/1-install-and-setup.md

```diff
@@ -103,8 +103,50 @@ You can optionally provide a custom FirebaseApp name with `initializeApp`.
 export class AppModule {}
 ```
 
+### 5. Setup individual @NgModules
 
-### 5. Inject AngularFireDatabase
+After adding the AngularFireModule you also need to add modules for the individual @NgModules that your application needs.  
+ - AngularFireAuthModule
+ - AngularFireDatabaseModule
+ - AngularFireStorageModule (Future release)
+ - AngularFireMessagingModule (Future release)
+
+#### Adding the Firebase Database and Auth Modules
+
+For example if you application was using both Firebase authentication and the Firebase database you would add:
+
+```ts
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { AppComponent } from './app.component';
+import { AngularFireModule } from 'angularfire2';
+import { AngularFireDatabaseModule } from 'angularfire2/database';
+import { AngularFireAuthModule } from 'angularfire2/auth';
+
+// Must export the config
+export const firebaseConfig = {
+  apiKey: '<your-key>',
+  authDomain: '<your-project-authdomain>',
+  databaseURL: '<your-database-URL>',
+  storageBucket: '<your-storage-bucket>',
+  messagingSenderId: '<your-messaging-sender-id>'
+};
+
+@NgModule({
+  imports: [
+    BrowserModule,
+    AngularFireModule.initializeApp(firebaseConfig, 'my-app-name'),
```

Comment:
Add a comment for each line saying what it adds in terms of Firebase, just so we can help those that hastily copy and paste.

```ts
AngularFireModule.initializeApp(firebaseConfig, 'my-app-name'), // imports firebase/app needed for everything
AngularFireDatabaseModule, // imports firebase/database, only needed for database features
AngularFireAuthModule, // imports firebase/auth, only needed for auth features
```